lib/compiler.js

```diff
@@ -451,17 +451,36 @@ function loadModelDefinition(rootDir, jsonFile, allFiles) {
 }
 
 function buildMiddlewareInstructions(rootDir, config) {
+
   var phasesNames = Object.keys(config);
   var middlewareList = [];
+
   phasesNames.forEach(function(phase) {
+
     var phaseConfig = config[phase];
+
     Object.keys(phaseConfig).forEach(function(middleware) {
+
       var allConfigs = phaseConfig[middleware];
       if (!Array.isArray(allConfigs))
         allConfigs = [allConfigs];
 
       allConfigs.forEach(function(config) {
-        var resolved = resolveMiddlewarePath(rootDir, middleware);
+
```

Comment:
Extra empty line, please remove.

In general, we try to avoid whitespace-only changes as they clutter the git history.